### PR TITLE
chore: Trivial cleanups/fixes/refactoring

### PR DESCRIPTION
**Pull request by UdjinM6**
## Issue being fixed or feature implemented
Trivial cleanups/fixes/refactoring

## What was done?
Pls see individual commits

## How Has This Been Tested?
Run tests

## Breaking Changes
n/a

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

File: src/validation.cpp [hunk 1]
```diff
@@ -4893,7 +4894,7 @@ bool CChainState::RollforwardBlock(const CBlockIndex* pindex, CCoinsViewCache& i
     // TODO: merge with ConnectBlock
     CBlock block;
     if (!ReadBlockFromDisk(block, pindex, m_params.GetConsensus())) {
-        return error("ReplayBlock(): ReadBlockFromDisk failed at %d, hash=%s", pindex->nHeight, pindex->GetBlockHash().ToString());
+        return error("RollforwardBlock(): ReadBlockFromDisk failed at %d, hash=%s", pindex->nHeight, pindex->GetBlockHash().ToString());
```
Comment (knst) [inline src/validation.cpp:4897]: maybe use __func__ for case if further refactorings?

```
-error("RollforwardBlock(): ReadBlockFromDisk failed at %d, hash=%s", pindex->nHeight, pindex->GetBlockHash().ToString());
+error("%s: ReadBlockFromDisk failed at %d, hash=%s", __func__, pindex->nHeight, pindex->GetBlockHash().ToString());
```

**Review by knst — APPROVED** [review]
utACK

**Review by ogabrielides — APPROVED** [review]
LGTM

**Pull request merged**